test/mocks/virtualHostAgent.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const nock = require('nock');
+const credentials = {
+  host: 'example.org:31415',
+  uri: 'mongodb://5e91c9e75a0aec0a4d8e9c3523887576:12b6ca434be2dd8716d1009eb280e1c5@10.244.10.160:27017,10.11.252.21:27017,10.11.252.22:27017/8354aac89a3781b8?replicaSet=282477f09a2704e7e1ab1396f7721acc'
```

Comment:
i know it does matter what you put in here as mock data, but it helps to have data which reflect the actual ones. Havent seen one though, but does it have a mongodb uri format for rabbit? if not, can you put in some mock which looks like a real one.